docs/sources/reference/components/pyroscope/pyroscope.receive_http.md

```diff
@@ -11,6 +11,10 @@ title: pyroscope.receive_http
 The HTTP API exposed is compatible with the Pyroscope [HTTP ingest API](https://grafana.com/docs/pyroscope/latest/configure-server/about-server-api/).
 This allows `pyroscope.receive_http` to act as a proxy for Pyroscope profiles, enabling flexible routing and distribution of profile data.
 
+{{< admonition type="note" >}}
+The `pyroscope.receive_http` component is currently in public preview. To use this component, you must run Alloy with the `--stability.level` flag set to `public-preview`. For more information about Alloy's run usage, refer to the [run command](https://grafana.com/docs/grafana-cloud/send-data/alloy/reference/cli/run/#the-run-command) documentation.
```

Comment:
We have a shared component for Public Preview in the Alloy repo: `alloy/docs/sources/shared/stability/public_preview.md`

Maybe it'd be better to use this element instead of the note?